### DISCUSSION TITLE
[Feature Request] Easy Digital Download Integration For WordPress

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#300] [Plugins & integrations]
Now that we have Lightning, it would be SICK if someone could tackle EDD integration. It's basically a WooCommerce alternative but for digital files. 

I noticed that Blockonomics has a [plugin for WordPress ](https://github.com/blockonomics/easy-digital-downloads-plugin)and it's open-source.

Leaving one more suggestion here that hopefully, we can tackle in the future.

**Comment by austrocoin** [comment]
I made not bad experience in outsource that "CMS plugin works" to some guys what do that for living. You have that affordable experts specially on that WP main plugins like EDD what make you that in no time.

Have now also a startup where I am invested what use shopify.... was thinking already same how to solve it :)

**Comment by pavlenex** [comment]
Closing this as it appears there's no much interest in it for now. Will re-open if there's community interest.

**Comment by pclweb** [comment]
With the great features now available in btcpayserver would you be able to create an easy digital downloads plugin. It's the only online shopping system not being served. I would love to remove the coinbase plugin and use a btcpayserver. I believe it's based on the bitpay api. I found a plugin on github that may help in porting it.

https://github.com/bitpay/bitpay-checkout-easy-digital-downloads

Thanks
Peter

**Comment by royserpa** [comment]
> 
> 
> Now that we have Lightning, it would be SICK if someone could tackle EDD integration. It's basically a WooCommerce alternative but for digital files.
> 
> I noticed that Blockonomics has a [plugin for WordPress ](https://github.com/blockonomics/easy-digital-downloads-plugin)and it's open-source.
> 
> Leaving one more suggestion here that hopefully, we can tackle in the future.

You can easily integrate this on your site.

Set a form and handle notifications from url or email methods. Have a script constantly check if invoice paid, and once paid, you show the download link to the user.

Hmmm... Nice way to set up a warez site lel!

**Comment by andres-pinilla** [comment]
+1

**Comment by PlebeiusGaragicus** [comment]
I've been online today trying to get digital downloads integration and I'm kind of upset at the lack of options.  I'm not sure why this hasn't been integrated into BTCPay Server already tbh - it is where I went first to try and find a solution.  Some of the solutions I have found require 'premium' features to enable and start at $40 a month (WordPress plugins, for example)

Not a good option for my use case - so I'm probably going to just make my own solution from scratch and wait for a more seasoned developer to tackle this.

**Comment by sicbitm** [comment]
Well, in 2024, we really need a BTCPay Server plugin for EDD!
